cypress/e2e/bookStore.cy.js

```diff
@@ -5,7 +5,36 @@ describe('Book Store app', () => {
     
   });
 
-  it('', () => {
-    
+it('Log in with valid credentials', () => {
+    cy.login('koko', 'Koko123*')
+    cy.visit('https://demoqa.com/login')
+    cy.contains('#item-2', 'Book Store').click();
+    cy.get('#searchBox').type('Speaking JavaScript');
+    cy.contains('.mr-2', 'Speaking JavaScript').click();
+    cy.contains('#addNewRecordButton', 'Add To Your Collection')
+      .click();
+    cy.on('window:alert', (str) => {
+        expect(str).to.equal(testData.alert.bookAdded);
+      });
+    cy.on('window:alert', (str) => {
+        expect(str).to.equal(`Book added to your collection.`)
+    cy.visit('https://demoqa.com/profile');
+    cy.contains('.ReactTable', 'Speaking JavaScript')
+        
+    })
+
```

Comment:
You didn't close brackets correctly in first test. So, only first test is running. Please check the second test. It's failed.